module/msg_smt/test/mod_msg_smt_unit_test.c

```diff
@@ -0,0 +1,815 @@
+/*
+ * Arm SCP/MCP Software
+ * Copyright (c) 2023, Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+
+#include "scp_unity.h"
+#include "unity.h"
+
+#include <mod_msg_smt.h>
+#include <mod_scmi.h>
+
+#include <Mockfwk_id.h>
+#include <Mockfwk_module.h>
+#include <Mockfwk_mm.h>
+
+#include UNIT_TEST_SRC
+#include "config_msg_smt.h"
+
+void setUp(void)
+{
```

Comment:
Could you use setUp and tearDown to initialize the module context?
similar comment in https://github.com/ARM-software/SCP-firmware/pull/806/files#r1310059185